Filters/exceptions.txt

```diff
@@ -21,6 +21,8 @@
 @@||f-gear.ec-optimizer.com^|
 !
 ! Blocked by CNAME unbouncepages.com
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/1826
+@@||get.neofinancial.com^|
```

Comment:
As far as I understand, it was blocked by CNAME `unbouncepages.com` which has been excluded in https://github.com/AdguardTeam/AdGuardSDNSFilter/commit/845ba72f03177950f05ad49b240f7d698f410403, so I guess that it's not needed now?

Reply:
I also see the exclusion, but it's still blocked for some reason.

Reply:
I guess that the problem is that rule `||unbouncepages.com^` is still in AdGuard DNS filter - https://raw.githubusercontent.com/AdguardTeam/FiltersRegistry/master/filters/filter_15_DnsFilter/filter.txt

Reply:
Yes, known problem.